sdk/rack/process.go

```diff
@@ -114,12 +111,13 @@ func (c *Client) ProcessRun(app string, opts types.ProcessRunOptions) (int, erro
 	defer r.Close()
 
 	var code int
+	var rerr string
 
-	if err := helpers.Stream(helpers.CodeGrabber(opts.Output, &code), r); err != nil {
+	if err := helpers.Stream(helpers.CodeGrabber(opts.Output, &code, &rerr), r); err != nil {
 		return 0, err
```

Comment:
Should these return non-zero here? Or will something else in `cx` take the error and return non-zero?

I'd like to have a convention for an exit code that means internal error, to help workflows differentiate between:

* command ran successfully
* command reported a failure, i.e. `cx run exit 1`
* command failed due to internal error

We can pick anything for that, how about `71`, borrowing from sysexits.h:

```
#define EX_OSERR        71      /* system error (e.g., can't fork) */
```

https://stackoverflow.com/questions/1101957/are-there-any-standard-exit-status-codes-in-linux